### DISCUSSION TITLE
Having issues using it with Sveltekit

### DISCUSSION THREAD

**Discussion by devbyVeriTech** [pockethost/pockethost#290] [Q&A]
Whenever I use the URL given to me, I get and error trying to read or create a collection but I've never gotten any issue when using the localhost URL

**Comment by benallfree** [comment]
Do you mean that your pocket base admin is not allowing you to create collections?

**Comment by devbyVeriTech** [comment]
![pb](https://github.com/pockethost/pockethost/assets/131156642/4dfac9bb-aacd-4205-b272-d8be17c2ea26)
![vs](https://github.com/pockethost/pockethost/assets/131156642/589e0b4a-a99e-4d73-a7e1-dc99c38d3ab9)

Here are some examples of the issue.
There are no logs sent and I get error report in my VS code terminal
